sparse/src/KokkosSparse_getDiagCopy.hpp

```diff
@@ -43,7 +43,8 @@
 */
 
 /// \file Kokkos_Sparse_getDiagCopy.hpp
-/// \brief Get a copy of the diagonal entries of a KokkosSparse::CrsMatrix.
+/// \brief Get a copy of the diagonal entries of a
```

Comment:
The line on the old version looks fine to mt.

Reply:
clang-format split it since CrsMatrix was CompressedSparseMatrix during the refactor